.travis.yml

```diff
@@ -25,6 +25,11 @@ script:
   - flake8 *.py tests ga4gh scripts --exclude=ez_setup.py
   - nosetests --with-coverage --cover-package ga4gh
               --cover-inclusive --cover-min-percentage 85
-              --cover-branches --cover-erase
+              --cover-branches --cover-erase -a '!travis_only'
```

Comment:
I don't think we need this, or to touch the test cases. We should be able to provoke the error with

```
pip install <tarball>
ga4gh_server --version
```
Does this work?